### PR TITLE
Add PSO deployment script

### LINKED ISSUE #201: StableToken - Create PSO proxy contract, deployment script and execute
**Description**

As the title says. Deploy the new PSO proxy across all networks and update the docs to include the new address.

**Acceptance Criteria**

- [ ] Create new proxy contract
- [ ] Deployment script is added to the repo
- [ ] Proxy contract is successfully deployed on all networks
- [ ] The new address is added to the docs

### PR DESCRIPTION
**Pull request by denviljclarke**
### Description

- Add & execute deployment scripts for StableTokenPSOProxy (PSO)

### Other changes

- None

### Tested

- N/A

### Related issues

- Fixes #201 

### Backwards compatibility

N/A

### Documentation
- N/A

### CODE REVIEW

**Review by bowd — APPROVED** [review]

File: .gitmodules [hunk 1]
```diff
@@ -16,3 +16,6 @@
 [submodule "lib/mento-core-2.3.1"]
 	path = lib/mento-core-2.3.1
 	url = https://github.com/mento-protocol/mento-core
+[submodule "lib/mento-core-2.4.0"]
```
Comment (philbow61) [inline .gitmodules:19]: why do we need the new submodule?
I thought we had a submodule pointing to develop. On which we can just pull the latest on? (2.3.1 maybe)

**Review by philbow61 — APPROVED** [review]
LGTM 👍

File: broadcast/PSO-00-Create-Proxies.sol/42220/run-latest.json [hunk 2]
```diff
@@ -0,0 +1,117 @@
+{
+  "transactions": [
+    {
+      "hash": "0x7d0088e4798fee338276bace188c9b44fbb290c6f6ca437e95c01d8d76b977a3",
+      "transactionType": "CREATE",
+      "contractName": "StableTokenPSOProxy",
+      "contractAddress": "0x7242a5FfbC765F322ee7cAd7957c291aC54c33F4",
```
Comment (chapati23) [inline broadcast/PSO-00-Create-Proxies.sol/42220/run-latest.json:7]: this is unverified on celoscan, not sure when/where we usually verify contracts but intuitively I would have expected verification to be part of this PR

**Review by chapati23 — COMMENTED** [review]
proxy still unverified on celoscan, should we verify it as part of this PR?

File: .gitmodules (hunk 1 above)
Comment (bowd) [reply .gitmodules:19]: We should keep submodules pointing to the version if we need a `develop` one it should be temporarily added as `mento-core-develop`, once we have a version tagged it should be pinned. And I think Jamal cut a version for `2.4.0`

File: broadcast/PSO-00-Create-Proxies.sol/42220/run-latest.json (hunk 2 above)
Comment (bowd) [reply broadcast/PSO-00-Create-Proxies.sol/42220/run-latest.json:7]: on it, will merge after 👍

**Pull request merged**